src/store/model-store.js

```diff
@@ -1,33 +1,26 @@
 import { Store } from 'consus-core/flux';
-
+import { searchModel } from '../lib/api-client';
 let model = null;
 let models = [];
 
 class ModelStore extends Store {
 
     getModel() {
-        if (model === null) {
-            return model;
-        }
-        return {
-            address: model.address,
-            name: model.name
-        };
+        return model;
```

Comment:
The current implementation returns a new object with the same attributes. Changing those attributes in the returned object would not change the original object.

Your change returns the original object. Changing the returned object would change the original object.

I'm not sure which one is better. The original is "correct" flux (you shouldn't be able to change any data in the store without creating a new action), but maybe we don't want that.